styles/_variables.scss

```diff
@@ -0,0 +1,8 @@
+@import url('https://fonts.googleapis.com/css2?family=Rajdhani:wght@300;500&display=swap');
+
```

Comment:
So glad you got these variables in here, that'll be great for a professional style throughout!